app/ui-react/syndesis/src/modules/support/pages/SupportPage.tsx

```diff
@@ -0,0 +1,19 @@
+import { PageSection, SupportHeader } from '@syndesis/ui';
+import * as React from 'react';
+import { PageTitle } from '../../../shared';
+
+export function SupportPage() {
+  return (
+    <>
+      <PageTitle title="Support" />
+      <SupportHeader
```

Comment:
We have a component to handle headers in the app, it's the `SimplePageHeader`. I'm replacing "custom" headers with it every time I can to be sure that we render the header always in the right way, in terms of text sizing and structure.